doc/man/lttng-relayd.8.txt

```diff
@@ -114,6 +115,9 @@ option:-o 'PATH', option:--output='PATH'::
 See the <<output-directory,Output directory>> section above for more
 information.
 
+option:-w 'PATH', option:--working-directory='PATH'::
+    Set the working directory of lttng-relayd processes.
```

Comment:
Let's use:

    Set the working directory of the processes this relay daemon creates.

Reply:
done